tests/torch/test_pytorch_patch.py

```diff
@@ -67,3 +70,22 @@ def forward(self, x: torch.Tensor):
 def test_jit_if_tracing_script_source():
     # Run test case in a separate process to track patching of torch by NNCF
     run_pytest_case_function_in_separate_process(test_jit_if_tracing_script_source_equals)
+
+
+def test_jit_script_signature():
```

Comment:
Can we also have a test for the model optimized with NNCF? Let's say we take a model from torchvision (MobilNetv2), apply quantization on a dummy dataset, call `.prepare_for_inference()`, and do the tracing/scripting then.

Reply:
Supporting scripting/tracing after `.prepare_for_inference()` is planned for another PR (ticket 106013). This one is only to fix the issue with `kornia` package.